Fix missing monsters loot, skinning corpses and npcs keywords

rigis1 opened the pull request:
# Description

+Zethra now is able to trade
+Gloom wolf now drops Gloom Wolf Fur
+Ghost wolf now drops gold coins
+Lizard legionnaire is now skinnable 
+Traveling cost is more accurate
+Additional chatlines 
+Twisted pooka drops more accurate loot
+Pixie drops more accurate loot
+Nymph drops more accurate loot
+Boogy drops more accurate loot
+Dragon and Grand Mother Foulscale now have corpse protection, skinning and autoloot.

## Behaviour
### **Actual**

Zethra not offer trade opportunity
You can't skin Lizard legionnaire corpse
Pino offer to travel to edron.

### **Expected**

Zethra now is able to trade, monsters drops more accurate loot and flying cost is as it should be.
Lizard legionnaire is now skinnable.
Pino flies only to Darashia, Svargrond, Femor Hills, Kazordoon, Zao and Issavi.

## Fixes
Resolves #612

## Type of change

Please delete options that are not relevant.

  - [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested
Go to Adventurers' Guild and start trade with Zethra
Kill Lizard legionnaire and use obsidian knife on its corpse.
Go to Pino and type Edron.
## Checklist

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I checked the PR checks reports
  - [x] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

MajestyOTBR commented:
Tested, working. Thanks for the fix.

rigis1 committed:
missing loot

gold

rigis1 committed:
missing Gloom Wolf Fur

rigis1 committed:
+lizard legionnaire skinning

rigis1 committed:
additional chatlines +travel cost adjust

rigis1 committed:
-edron destination +adjust cost travel

rigis1 committed:
+additional chatlines

rigis1 committed:
adjust travel cost

rigis1 committed:
adjust travel cost

rigis1 committed:
+missing loot and adjust ancient coin maxcount

rigis1 committed:
+missing loot

rigis1 committed:
too much loot

rigis1 committed:
+missing loot

rigis1 committed:
+trade option missing

rigis1 committed:
fix corpse id

rigis1 committed:
adjust speed and armor

rigis1 committed:
fazzrah teleport fix

guispiller commented on data/monster/fey/boogy.lua:
```diff
@@ -95,6 +95,9 @@ monster.loot = {
 	{id = 814, chance = 3200}, -- terra amulet
+	{name = "Fairy Wings", chance = 15000},
+	{name = "Wooden Spellbook", chance = 3440},
+	{name = "Leaf Star", chance = 3440, maxCount = 7},
```
Could you please just fix these 3 lines to keep the standard of all other monsters? If the loot is registered by the name, please use all letters with lower case. Thanks for your contribution

guispiller commented on data/monster/fey/twisted_pooka.lua:
```diff
@@ -94,6 +94,8 @@ monster.loot = {
 	{name = "orange mushroom", chance = 5155},
 	{name = "shimmering beetles", chance = 591},
+	{name = "Prismatic Quartz", chance = 2500},
+	{name = "Crystal of Focus", chance = 60},
```
Could you please just fix these 2 lines to keep the standard of all other monsters? If the loot is registered by the name, please use all letters with lower case. Thanks for your contribution.

rigis1 committed:
fix to fazzrah ruin other teleports 

fazzrah teleport is still broken

rigis1 committed:
update to previous changes

rigis1 committed:
update to previous changes

rigis1 committed:
adjust speed and armor

rigis1 committed:
Correct corpse with protection

rigis1 committed:
Update to skinning dragons

dragon and grand mother foulscale skinning update

rigis1 committed:
Correction to dragons

dragon and grand mother foulscale now have corpse protection, skinning and autoloot

rigis1 committed:
Fix minotaur skinning

majestyotbr committed:
Fix trailing whitespace

guispiller reviewed (approved):
Nice job. Thanks for your contribution!
